packages/pds/src/db/migrations/20221116T234458063Z-duplicate-records.ts

```diff
@@ -0,0 +1,195 @@
+import { Kysely } from 'kysely'
+import DatabaseSchema from '../database-schema'
+import { Assertion } from '../tables/assertion'
+import { Follow } from '../tables/follow'
+import { Repost } from '../tables/repost'
+import { Trend } from '../tables/trend'
+import { Vote } from '../tables/vote'
+
+type DatabaseSchemaWithTemps = DatabaseSchema & {
+  repost_temp: Repost
+  trend_temp: Trend
+  vote_temp: Vote
+  follow_temp: Follow
+  assertion_temp: Assertion
+}
+
+const duplicateRecordTable = 'duplicate_record'
+
+export async function up(db: Kysely<DatabaseSchemaWithTemps>): Promise<void> {
+  await db.schema
+    .createTable(duplicateRecordTable)
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('duplicateOf', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .execute()
+
+  const duplicateReposts = await db
```

Comment:
@devinivy knows, but these are so long-winded because sqlite doesn't allow adding a constraint to an existing table 🙃